### DISCUSSION TITLE
Unable to create bucketclass

### DISCUSSION THREAD

**Discussion by dkoci** [noobaa/noobaa-core#6835] [Q&A]
Hi. I am learning noobaa. Can someone please explain me what are 'backingstore' and 'bucketclass'? I have deployed noobaa operator, connected to our private s3 storage and also created a bucket.
I need some explanation about backingstore and bucketclass. I have read README.md and tried to create backingstore and bucketclass. First one was successful, but I cant create bucketclass with existing backingstore:

![image](https://user-images.githubusercontent.com/5565897/146652025-daac3851-4914-434d-bfe7-e8840789cc9a.png)

Tnx

**Comment by jackyalbo** [comment]
@dkoci Just remove the square brackets. I can see how you got confused by the help... sorry for that.  for more than one backingstore you can just add commas - but not brackets are needed.

**Reply by dkoci** [reply]
It works. Thanks again.
Is there any difference between creating OBC in webconsole (mgmt gui) and noobaa cli? I created OBC with noobaa cli, then deleted it in mgmt gui but `noobaa obc list` still gives me the obc which I deleted in gui.
